examples/multi_arch/BUILD.bazel

```diff
@@ -30,3 +31,17 @@ oci_image_index(
         ":images",
     ],
 )
+
+genrule(
+    name = "hash",
+    srcs = [":index"],
+    outs = ["sha256.sum"],
+    cmd = "$(JQ_BIN) -r '.manifests[0].digest' $(location :index)/index.json > $@",
```

Comment:
nit: maybe you want `assert_json_matches` instead?

Reply:
I wanted to but it uses diff_test from skylib which doesn't support passing individual files out of tree artifacts.